content/docs/sidebar.json

```diff
@@ -10,7 +10,7 @@
     "source": "install/index.md",
     "children": [
       {
-        "label": "Mac OS",
+        "label": "macOS",
```

Comment:
```suggestion
        "label": "MacOS",
```

Reply:
macOS is the official name for the OS (like FreeBSD would not be Freebsd for consistency). I think logo being the same is just a coincidence. 

> Formally speaking any proper name starts with a capital letter

brand names are written the way the are written I think. We can refer to the Wiki page - https://en.wikipedia.org/wiki/MacOS - they use macOS everywhere.

Reply:
Sent from an Iphone 😋